velox/substrait/README.md

```diff
@@ -0,0 +1,8 @@
+# Velox substrait upgrade
+
+Velox copy substrait proto files from `https://github.com/substrait-io/substrait/tree/main/proto/substrait` to generate substrait cpp files. So if you want to upgrade substrait to a certain version, follow the steps like below:
+
+- Copy substrait proto files for a certain version in `https://github.com/substrait-io/substrait/tree/main/proto/substrait` to velox substrait proto dir: `https://github.com/facebookincubator/velox/tree/main/velox/substrait/proto/substrait`
+- Add `option cc_enable_arenas = true;` to the top of option definition area for every proto file
```

Comment:
Can we create a GitHub issue in Substrait project to ask if there is a way to get a copy of the file with cc_enable_arenas enabled without having to manually edit the files every time? Manual editing is error prone.

Reply:
If you are using a new enough version of protoc there is no need for `cc_enable_arenas`.  See https://github.com/substrait-io/substrait/pull/243 and https://github.com/protocolbuffers/protobuf/issues/8119